docs/source-app/workflows/byoc/index.rst

```diff
@@ -78,18 +78,14 @@ Parameters
 +------------------------+----------------------------------------------------------------------------------------------------+
 | region                 | AWS region containing compute resources                                                            |
 +------------------------+----------------------------------------------------------------------------------------------------+
-| instance-types         | Instance types that you want to support, for computer jobs within the cluster.                     |
-|                        |                                                                                                    |
-|                        | For now, this is the AWS instance types supported by the cluster.                                  |
-+------------------------+----------------------------------------------------------------------------------------------------+
 | enable-performance     | Specifies if the cluster uses cost savings mode.                                                   |
 |                        |                                                                                                    |
 |                        | In cost saving mode the number of compute nodes is reduced to one, reducing the cost for clusters  |
 |                        | with low utilization.                                                                              |
 +------------------------+----------------------------------------------------------------------------------------------------+
 | edit-before-creation   | Enables interactive editing of requests before submitting it to Lightning AI.                      |
 +------------------------+----------------------------------------------------------------------------------------------------+
-| wait                   | Waits for the cluster to be in a RUNNING state. Only use this for debugging.                       |
```

Comment:
How will this change affect @nicolai86  e2e BYOC CI work? IIRC the CI job uses --wait flag

Reply:
I will be picking up BYOC CI. I'll ensure it does not break on this.